README.md

```diff
@@ -1,10 +1,8 @@
 # simplesamlphp-module-assurance
 A SimpleSAMLphp module for determining and indicating the Level of Assurance (LoA) of an authentication event.
 Specifically the module determines the LoA baced on:
-* `candidates`, if the represented LoA attribute of the authenticating user contains any value of the allowed values, then the module will skip execution.
+* `assuranceMap`, if the represented LoA attributes, of the authenticating user, contain any of the list of the allowed values, then the module will append these values into the State.
```

Comment:
`attributeMap`, a map whose keys identify attribute names whose values can be mapped to assurance values.  represented LoA attributes, of the authenticating user, contain any of the list of the allowed values, then the module will append these values into the State.
Defaults to XXX

Reply:
Done